src/helpers.ts

```diff
@@ -1,5 +1,17 @@
 export const delay = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms));
 
+// TODO Reuse it in Select
+// TODO Use TextField component for select tests
+export const createFormFieldFilters = <E extends HTMLInputElement | HTMLSelectElement | HTMLTextAreaElement>() => ({
+  valid: (element: E) => !element.labels?.[0].classList.contains("Mui-error"),
+  required: (element: E) => element.required,
+  description: (element: E) => {
+    const descriptionId = element.getAttribute("aria-describedby");
+    const descriptionElement = descriptionId ? element.ownerDocument.getElementById(descriptionId) : null;
+    return descriptionElement?.innerText ?? "";
+  },
+});
+
```

Comment:
Let's put this is a `form-field-filters.ts`

Reply:
Wait for merge #14

Reply:
Fixed